nodes/pods/run_once_duration_override/run-once-duration-override-release-notes.adoc

```diff
@@ -12,5 +12,48 @@ To apply the run-once duration override from the {run-once-operator} to run-once
 
 These release notes track the development of the {run-once-operator} for {product-title}.
 
-:operator-name: The {run-once-operator}
-include::snippets/operator-not-available.adoc[]
\ No newline at end of file
+For an overview of the {run-once-operator}, see xref:../../../nodes/pods/run_once_duration_override/index.adoc#run-once-about_run-once-duration-override-about[About the {run-once-operator}].
+
+[id="run-once-duration-override-operator-release-notes-1-1-0"]
+== {run-once-operator} 1.1.0
+
+Issued: 2024-02-28
+
+The following advisory is available for the {run-once-operator} 1.1.0:
+
+* link:https://access.redhat.com/errata/RHSA-2024:0269[RHSA-2024:0269]
+
+[id="run-once-duration-override-operator-1-1-0-bug-fixes"]
+=== Bug fixes
+
+* This release of the {run-once-operator} addresses several Common Vulnerabilities and Exposures (CVEs).
+
+[id="run-once-duration-override-operator-release-notes-1-0-1"]
```

Comment:
@cbippley one quick question, would not just adding about 1.1.0 be sufficient here ? Do we need to add about all releases in this release notes ?

Reply:
You are correct, I have deleted them.

Reply:
@kasturinarra Does everything else look good?

Reply:
@cbippley still see 1.0.0, is that needed ?